README.md

```diff
@@ -2,23 +2,61 @@
 
 > The night is short, walk on girl.
 
-http://short-night.pea3nut.org 
+http://short-night.pea3nut.org
 
-:deciduous_tree: An engine for drawing timeline graph.
+:deciduous_tree: An engine for drawing timeline graphs.
 
-# What is project doing?
+# What is it?
 
-An engine for draw a timeline graph:
+An engine for drawing timeline graphs on canvas.
 
-- Which just given tiny info of events
-- Which has directly proportional between time and space.
-- Which can automatic fix the overlap
+-   Minimal events data as input
+-   Draws a timeline with axis, markers and event boxes
+-   Fully customizable colors, styles, fonts
+-   Time between events is directly proportional to distance on axis
+-   Automatically handles overlap of event boxes on timeline
 
 ![](result-demo.jpg)
 
-# Documents
+# Use cases
 
-Wanna using a timeline? click [here](http://short-night.pea3nut.org/example)
+-   Timeline of career events to put in a CV or resume to impress your employer
+-   Timeline of past events in history
+-   Share a task timeline of a project with a client
 
-Wanna to develop a new timeline? click [here](https://github.com/FoXZilla/short-night/wiki)
+# Demo
 
+Check out a demo [here](http://short-night.pea3nut.org/example)
+
+# Develop
```

Comment:
There are 2 scenarios that are needed to document:

1. Create a timeline based in Short-night
2. Development on short-night itself

For the first one, we don't need to use `npm link`. Because the developer doesn't need to adjust the source code

I think we should split 2 scenarios and document then separately

Reply:
The first scenario is actually better described under a **Usage** section. But since usage is quite complex and requires quite a few files (Event and Axis classes, etc), it's not appropriate in my opinion to put this in a single Readme. Rather we can make a **Usage** section that just refers the reader to the **Wiki** section isntead.